### PR TITLE
Add PS3 patching guide, fill out how-to-connect channel

### PR DESCRIPTION
**Pull request by jvyden**

### CODE REVIEW

File: patching/ps3.md
Comment (speedyman09) [inline patching/ps3.md:1]: A few corrections:

File: patching/ps3.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Using Refresher to patch a game on PS3
+
+Refresher has native support for decrypting, encrypting and patching PS3 games. This guide aims to show you how to operate Refresher's PS3 patcher.
+
+It also assumes that you've already downloaded Refresher. If you haven't head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
```
Comment (speedyman09) [inline patching/ps3.md:5]: add a comma after "haven't"

File: patching/ps3.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+It also assumes that you've already downloaded Refresher. If you haven't, head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
+
+![A Screenshot of Refresher's PS3 patching screen](https://littlebigrefresh.github.io/Docs/patching/images/refresher-ps3.png)
+
+## Step 1: Jailbreak your PS3 and install webMAN
+
+We can't help you with this, but there are various guides out there you can go out and find. We personally recommend MrMario2011's guides.
+
+In general, though, try to use latest firmware, use HEN for super slims, CFW for other models, etc., etc.
```
Comment (speedyman09) [inline patching/ps3.md:13]: "the" should be before "latest firmware"

**Review by speedyman08 — CHANGES_REQUESTED** [review]
A few changes

File: patching/ps3.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+## Step 3: Find your PS3's IP Address
+
+Head on over to your PS3, head to settings, then network settings, and open 'Settings and Connection Status List'.
+
+![A screenshot of the PS3, showing the mentioned menu.](https://littlebigrefresh.github.io/Docs/patching/images/ps3-ip.png)
+
+Look for your 'IP Address', and write it down under the 'PS3s IP' field in Refresher.
+Click away from the textbox, and Refresher should start connecting. It may take a couple seconds for it to download all the game icons and metadata.
+
+## Step 3: Fill in the server URL
```
Comment (speedyman08) [inline patching/ps3.md:28]: You have two step 3's

File: patching/ps3.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Drop the installer .pkg on your PS3, and you should be all set after you run it.
+
+## Step 3: Find your PS3's IP Address
+
+Head on over to your PS3, head to settings, then network settings, and open 'Settings and Connection Status List'.
+
+![A screenshot of the PS3, showing the mentioned menu.](https://littlebigrefresh.github.io/Docs/patching/images/ps3-ip.png)
+
+Look for your 'IP Address', and write it down under the 'PS3s IP' field in Refresher.
+Click away from the textbox, and Refresher should start connecting. It may take a couple seconds for it to download all the game icons and metadata.
```
Comment (speedyman08) [inline patching/ps3.md]: it should be "couple of"

File: patching/ps3.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+Most servers based on Bunkum should support this protocol, most notably being Refresh. Other servers such as Project Lighthouse do not support this protocol.
+
+If successful, Refresher will open a popup presenting the information it gathered from the server. Click 'Yes' to accept these settings, and Refresher will automatically do the rest when you patch.
+
+![A dialog box from Refresher, indicating that Autodiscover has been performed sucessfully.](https://littlebigrefresh.github.io/Docs/patching/images/autodiscover-success.png)
+
+> **Note**
+> If you are a developer of a custom server for any game, you can read up on our Autodiscover protocol's documentation [here](https://littlebigrefresh.github.io/Docs/autodiscover-api).
+>
+> We recommend you support it, as it helps users ensure that they are doing the correct thing before they launch the game.
```
Comment (speedyman08) [inline patching/ps3.md]: "doing the correct thing" should be replaced with "following the correct procedures"

File: patching/ps3.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+Look for your 'IP Address', and write it down under the 'PS3s IP' field in Refresher.
+Click away from the textbox, and Refresher should start connecting. It may take a couple seconds for it to download all the game icons and metadata.
+
+## Step 3: Fill in the server URL
+
+You should have been given a URL by the person operating the server you're trying to connect to. Copy this URL into the second field - the one named 'Server URL'.
+
+Next, hit the button that says 'Autodiscover'. This will trigger Refresher to attempt to connect to the remote server to gather more details and extra patching instructions from it.
+
+Most servers based on Bunkum should support this protocol, most notably being Refresh. Other servers such as Project Lighthouse do not support this protocol.
```
Comment (speedyman08) [inline patching/ps3.md]: Change the sentence here to "Most servers based on Bunkum should support this protocol, most notably being Refresh."

**Review by speedyman08 — CHANGES_REQUESTED** [review]
A few changes

File: patching/ps3.md (hunk 1 above)
Comment (speedyman08) [inline patching/ps3.md]: Forgot this here, comma after "haven't"

File: patching/ps3.md (hunk 2 above)
Comment (speedyman08) [inline patching/ps3.md]: there should be a "the" before "latest firmware"

File: patching/ps3.md (hunk 6 above)
Comment (jvyden) [reply patching/ps3.md]: That's exactly what it already says

Comment (speedyman08) [reply patching/ps3.md]: Fuck, i meant "Most servers based on Bunkum, most notably Refresh, should support this protocol."

**Comment by jvyden** [comment]
fuck it good enough

**Comment by speedyman08** [comment]
This is truly the best documentation of All Time

**Pull request merged**